corehq/apps/sms/templates/sms/languages.html

```diff
@@ -20,6 +20,7 @@
 {% block js %}{{ block.super }}
     <script src="{% static 'hqwebapp/js/lib/jquery-ui/jquery-ui-1.8.16.min.js' %}"></script>
     <script src="{% static 'style/ko/knockout_bindings.ko.js' %}"></script>
+    <script src="{% static 'hqwebapp/js/ajax_csrf_setup.js' %}"></script>
```

Comment:
This seems like a hard problem where this file is now a dependency for `main.js`... after talking to @sravfeyn sounds like this is a tradeoff between one form of DRYness with another.